docs/docs/deployment.md

```diff
@@ -193,6 +206,49 @@ cfg := &tls.Config{
 </TabItem>
 </Tabs>
 
+### Using GPUs
+
+If the cluster is [configured for GPU usage](./getting-started/bare-metal.md#preparing-a-cluster-for-gpu-usage), Pods can optionally be facilitated with GPU devices.
+
+To do so, a CDI annotation needs to be added, specifying to use the `pgpu` (**p**assthrough **GPU**) mode. The `0` corresponds to the PCI device index. For nodes with a single GPU, this should always be `0`.
+For nodes with more than one GPU, this needs to be adjusted by the order the devices are enumerated in on the PCI bus. This can be found by checking the `/var/run/cdi/nvidia.com-pgpu.yaml` file on the node in question.
+
+As the footprint of a GPU-enabled pod-VM is larger than one of a non-GPU one, the memory of the pod-VM can be adjusted by using the `io.katacontainers.config.hypervisor.default_memory` annotation, which receives the memory the
+VM should receive in MiB. The example below sets it to 16 GB.
+
+```yaml
+metadata:
+  # ...
+  annotations:
+    # ...
+    cdi.k8s.io/gpu: "nvidia.com/pgpu=0"
+    io.katacontainers.config.hypervisor.default_memory: "16384"
+```
+
+Additionally, the container within the pod that should receive the GPU device needs to have a device request, specifying how many of each GPU the workload should receive. Here, the identifiers obtained when
+[deploying the NVIDIA GPU Operator](./getting-started/bare-metal.md#preparing-a-cluster-for-gpu-usage) need to be used. The example facilitates the container with a single NVIDIA H100 GPU.
+
+Finally, the environment variable `NVIDIA_VISIBLE_DEVICES` needs to be set to `all` so that the container is facilitated with additional GPU utilities from the pod-VM, such as the CUDA libraries required to
+run GPU workloads.
```

Comment:
```suggestion
Finally, the environment variable `NVIDIA_VISIBLE_DEVICES` must be set to `all` to grant the container access to GPU utilities provided by the pod-VM. This includes essential tools like CUDA libraries, which are required for running GPU workloads effectively.
```

Reply:
I cut out "effectively" here, as it's required for virtually any GPU workload.